ONSdigital/rm-collection-exercise-service
# Remove redis from collection exercise

# What and why?

During the java 17 upgrade, we noticed the redisson package needed updating.  After thinking about it, we might not need redis in collection exercise as that was probably used as part of the old style of sample load/case creation.  This PR will remove all mentions of it to make and then we can test that everything still works as expected.

# How to test?

- Deploy environment with this PR in it
- Either
  - Run acceptance tests (to set up an exercise and have someone sign up for it), or
  - Create a collection exercise in your environment, set it live and enrol on it.
- After this, check response ops pages to make sure nothing breaks.  Verify the response chasing report still works
- Also check frontstage as a respondent.

Nothing should be different than it normally is as everything that was deleted should've only been used by the old sample load system.  Also, now that we use pubsub, that rabbitmq config (in the 'messaging' key) should be fine to delete as well.


# Trello
